app/scripts/components/exploration/data-utils.ts

```diff
@@ -198,8 +198,13 @@ export function resolveRenderParams(
   // Return user defined source parameters if there is one
   if (datasetSourceParams?.assets) return rest;
   // 3. Return dashboard namespace render extension data
-  if (queryDataRenders && queryDataRenders[RENDER_KEY])
+  if (
+    queryDataRenders &&
+    queryDataRenders[RENDER_KEY] &&
+    !Array.isArray(queryDataRenders[RENDER_KEY].bidx)
```

Comment:
I made a small change because bidx is always array (even with one bidx.) - I changed it to return user defined sourceparams when asset_bidx is defined, which also should be revisited.